main.js

```diff
@@ -1,17 +1,17 @@
 var $car = document.getElementById('car');
-console.log($car);
+var direction;
 document.addEventListener('keydown', function (event) {
-  if (event.code === 'ArrowUp') {
-    $car.style.transform = "rotate(270deg)"
+  if (event.code === 'ArrowUp' && direction !== 'up') {
+    direction = 'up';
   }
-  else if (event.code === 'ArrowDown') {
-    $car.style.transform = "rotate(90deg)"
+  else if (event.code === 'ArrowDown' && direction !== 'down') {
+    direction = 'down';
   }
-  else if (event.code === 'ArrowLeft') {
-    $car.style.transform = "rotate(180deg)"
+  else if (event.code === 'ArrowLeft' && direction !== 'left') {
+    direction = 'left';
   }
-  else if (event.code === 'ArrowRight') {
-    $car.style.transform = "rotate(0deg)"
+  else if (event.code === 'ArrowRight' && direction !== 'right') {
```

Comment:
No need to make sure if the direction is not the same value, you can update it still, it wont make a difference.